VictoryCenter/VictoryCenter.WebAPI/.env.example

```
@@ -1,10 +1,10 @@
 ﻿DB_CONNECTION_STRING="server=localhost;Database=VictoryCenterDb;User Id=sa;Password=Admin@1234;MultipleActiveResultSets=true;TrustServerCertificate=true;Encrypt=false;"
 INITIAL_ADMIN_EMAIL=admintest@gmail.com
 INITIAL_ADMIN_PASSWORD=P123w0rd!
-JWTOPTIONS_SECRETKEY=DD8FF9F7-523A-41DB-8648-F125A714DE29
-JWTOPTIONS_REFRESH_TOKEN_SECRETKEY=8253B262-E2F9-4DCE-B044-B2D9999E1785
+JWTOPTIONS_SECRETKEY=A1B2C3D4-E5F6-4A7B-9C0D-123456789ABC
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWTOPTIONS_SECRETKEY key should go before the ﻿DB_CONNECTION_STRING key

(UnorderedKey)